frontend/src/js/snack-message/SnackMessage.tsx

```diff
@@ -7,13 +7,20 @@ import { useClickOutside } from "../common/helpers/useClickOutside";
 import FaIcon from "../icon/FaIcon";
 
 import { setMessage } from "./actions";
+import { SnackMessageStateT, SnackMessageTypeT } from "./reducer";
 
-const Root = styled("div")`
+const colorLookupTable: { [key in SnackMessageTypeT]: string } = {
```

Comment:
Let's avoid calling variables by their "data structure" ("lookup table") and instead describe what they contain. Also `Record` is useful here:
```suggestion
const bgColorByType: Record<SnackMessageType, string> = {
```